main.yml

```diff
@@ -20,8 +20,14 @@
 
   - debug: msg="{{ vm_ip.stdout }}"
 
+  - name: Get username
+    shell: cat /etc/passwd | grep "home" | cut -d {{':'}} -f 1
+    register: username
+
+  - debug: msg="{{ username.stdout }}"
```

Comment:
We can remove `debug` statement actually. But `whoami` would be `root` user in this case.